pygmt/src/coast.py

```diff
@@ -66,12 +66,11 @@ def coast(self, **kwargs):
     {J}
     {R}
     area_thresh : int, float, or str
-        *min_area*\ [/*min_level*/*max_level*][**+ag**\|\ **i**\
-        \|\ **s**\|\ **S**][**+r**\|\ **l**][**+p**\
-        *percent*].
-        Features with an area smaller than min_area in km^2 or of
-        hierarchical level that is lower than min_level or higher than
-        max_level will not be plotted.
+        *min_area*\ [/*min_level*/*max_level*][**+a**\[**g**\|\ **i**]
+        [**s**\|\ **S**][**+l**\|\ **r**][**+p**\ *percent*].
+        Features with an area smaller than min_area in km\ :sup:`2` or of
```

Comment:
Another way to write <code>km\ :sup:`2`</code> is <code>:math:`km^2`</code>. I'm not sure which one is better.